evap/staff/tests/test_views.py

```diff
@@ -222,6 +222,67 @@ def test_suspicious_operation(self):
         self.assertEqual(reply.status_code, 400)
 
 
+class TestSemesterCourseImportContributorsView(ViewTest):
+    url = "/staff/semester/1/course/1/contributor_import"
+    test_users = ["staff"]
+    filename_valid = os.path.join(settings.BASE_DIR, "staff/fixtures/valid_user_import.xls")
+    filename_invalid = os.path.join(settings.BASE_DIR, "staff/fixtures/invalid_user_import.xls")
+
+    @classmethod
+    def setUpTestData(cls):
+        semester = mommy.make(Semester, pk=1)
+        mommy.make(UserProfile, username="staff", groups=[Group.objects.get(name="Staff")])
+        cls.course = mommy.make(Course, pk=1, semester=semester)
```

Comment:
@karyon I'm not entirely sure wether this is what you meant, so please have look at it again 🙂